### DISCUSSION TITLE
Extend main logger to use default metadata

### DISCUSSION THREAD

**Discussion by JohnHillegass** [directus/directus#9894] [Feature Requests]
It would be great to be able to extend the main logger using a mixin or a child logger to add default metadata to the log outputs. This would be helpful when ingesting logs into Splunk or something similar from a shared compute environment/log grouping. Currently I see a path forward with changing it via extension context a la #7777 but not a clear path for applying default metadata to all logs. 

https://getpino.io/#/docs/api?id=mixin-function 
https://getpino.io/#/docs/child-loggers

**Comment by JohnHillegass** [comment]
Related to https://github.com/directus/directus/discussions/8643

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
